src/Numerics/ODESolvers/SplitExplicitMethod.jl

```diff
@@ -0,0 +1,192 @@
+export SplitExplicitSolver
+
+using ClimateMachine.DGMethods:
+    initialize_states!,
+    tendency_from_slow_to_fast!,
+    cummulate_fast_solution!,
+    reconcile_from_fast_to_slow!
+
+LSRK2N = LowStorageRungeKutta2N
+
+"""
```

Comment:
Since this is a custom / home baked stepper might be worth documenting mathematically what it's doing. (Just thinking that our future selves will need to understand it, and we will forget.)

Reply:
Am I correct that this is a very simple first order splitting of the ODE?
```
dQ = f(Q,P)
dP = g(Q, P)
```
and you are doing something like:
```
dQ = f(Q, P^n)
```
to take `Q^n` to `Q^{n+1} and then
```
dP = g(Q^{n+1}, P)
```
to take `P^n` to `P^{N+1}`?

Reply:
this sounds about right, @sandreza and @jm-c can you comment further?

Reply:
the latest commit 287c82149076e0593cca1c2a333b6c990642adb9 shows the details of the splitting by filling in the communication functions. there's still some work needed before this can be used, because we need to add two auxiliary balance laws in order to do the vertical integrals needed, but the layout of what is happening is there.

Reply:
will address in #1268